chart/Chart.yaml

```diff
@@ -107,14 +107,14 @@ annotations:
     - name: prometheus-config-reloader
       image: quay.io/prometheus-operator/prometheus-config-reloader:v0.81.0
     - name: csi-attacher
-      image: registry.k8s.io/sig-storage/csi-attacher:v4.3.0
+      image: registry.k8s.io/sig-storage/csi-attacher:v4.8.1
     - name: csi-node-driver-registrar
-      image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.10.0
+      image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.13.0
     - name: csi-provisioner
-      image: registry.k8s.io/sig-storage/csi-provisioner:v3.5.0
+      image: registry.k8s.io/sig-storage/csi-provisioner:v5.2.0
     - name: csi-resizer
-      image: registry.k8s.io/sig-storage/csi-resizer:v1.9.3
+      image: registry.k8s.io/sig-storage/csi-resizer:v1.13.2
     - name: csi-snapshotter
-      image: registry.k8s.io/sig-storage/csi-snapshotter:v6.3.3
+      image: registry.k8s.io/sig-storage/csi-snapshotter:v8.2.0
     - name: snapshot-controller
-      image: registry.k8s.io/sig-storage/snapshot-controller:v6.3.3
+      image: registry.k8s.io/sig-storage/snapshot-controller:v8.2.0
```

Comment:
Are we using the same versions across zfs-localpv and lvm-localpv?

Reply:
Yes